README.md

```diff
@@ -1,205 +1,95 @@
-    <img src="https://platform.sh/images/deploy/lg-blue.svg" alt="Deploy on Platform.sh" width="180px" />
-</a>
-
-#### APT (Debian & Ubuntu)
-
-```bash
-echo "deb [trusted=yes] https://apt.fury.io/meilisearch/ /" > /etc/apt/sources.list.d/fury.list
-apt update && apt install meilisearch-http
-meilisearch
-```
-
-#### Download the binary (Linux & Mac OS)
-
-```bash
-curl -L https://install.meilisearch.com | sh
-./meilisearch
-```
-
-#### Compile and run it from sources
-
-If you have the latest stable Rust toolchain installed on your local system, clone the repository and change it to your working directory.
-
-```bash
-git clone https://github.com/meilisearch/meilisearch.git
-cd meilisearch
-cargo run --release
-```
-
-### Create an Index and Upload Some Documents
-
-Let's create an index! If you need a sample dataset, use [this movie database](https://www.notion.so/meilisearch/A-movies-dataset-to-test-Meili-1cbf7c9cfa4247249c40edfa22d7ca87#b5ae399b81834705ba5420ac70358a65). You can also find it in the `datasets/` directory.
-
-```bash
-curl -L https://docs.meilisearch.com/movies.json -o movies.json
-```
-
-Now, you're ready to index some data.
-
-```bash
-curl -i -X POST 'http://127.0.0.1:7700/indexes/movies/documents' \
-  --header 'content-type: application/json' \
-  --data-binary @movies.json
-```
-
-### Search for Documents
-
-#### In command line
-
-The search engine is now aware of your documents and can serve those via an HTTP server.
+## ✨ Features
 
-The [`jq` command-line tool](https://stedolan.github.io/jq/) can greatly help you read the server responses.
+- **Search-as-you-type:** relevant search results in less than 50 milliseconds
+- **[Typo tolerance](https://docs.meilisearch.com/learn/getting_started/customizing_relevancy.html#typo-tolerance):** Meilisearch adapts to typos and misspellings so your search results are always relevant
+- **[Filtering and faceted search](https://docs.meilisearch.com/learn/advanced/filtering_and_faceted_search.html#configuring-filters):** enhance your user's search experience with custom filters and build a faceted search interface in a few lines of code
+- **[Sorting](https://docs.meilisearch.com/learn/advanced/sorting.html):** sort results based on price, date, or pretty much anything else your users need
+- **[Synonym support](https://docs.meilisearch.com/learn/getting_started/customizing_relevancy.html#synonyms):** configure synonyms to include more relevant content in your search results
+- **[Geosearch](https://docs.meilisearch.com/learn/advanced/geosearch.html):** filter and sort documents based on geographic data
+- **[Extensive language support](https://docs.meilisearch.com/learn/what_is_meilisearch/language.html):** Meilisearch works with all languages and offers optimized support for languages using the Latin alphabet, Chinese, Japanese, and Hebrew
```

Comment:
That's an interesting point. I do not read "optimized support" as necessarily referring to performance improvements and as "these languages will work well because we're specifically targeting them," but this might be my own bias.

If we change it here, we should also change it in the docs, as [we speak about "optimized support" straight away](https://docs.meilisearch.com/learn/what_is_meilisearch/language.html), in our first sentence 😅 

Curious to hear other people's thought on this.

Reply:
Same I don't think about performance when I read "optimized", but not the best in English here